quantmsrescore/ms2rescore.py

```
@@ -398,136 +269,85 @@
-    if rescoring_engine == "moakapot":
-        logging.warning(
-            "Mokapot rescoring engine is not supported in this version. Please use Percolator."
-        )
-        raise ValueError(
-            "Mokapot rescoring engine is not supported in this version. Please use Percolator."
-        )
-
-    config = parse_cli_arguments_to_config(
-        config_file=config_file,
-        output_path=output_path,
+    annotator = Annotator(
```

Ruff (0.8.2)

Local variable `annotator` is assigned to but never used

Remove assignment to unused variable `annotator`

(F841)